### PR TITLE
Homework12: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**
homework_12_OOP

### CODE REVIEW

**Comment by ETsiunel** [comment]
I temporally disabled mypy linter

**Comment by Stml89** [comment]
> I temporally disabled mypy linter

oh, yes. I was busy

File: hw_12/1.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    def __init__(self):
+        self.cards = self.create_deck()
+
+    def create_deck(self):
+        deck = []
+        for mast in Card.mast_list[:-1]:
+            for number in Card.number_list:
+                deck.append(Card(number, mast))
+        deck.append(Card('Joker', Card.mast_list[-1]))
+        deck.append(Card('Joker', Card.mast_list[-1]))
```
Comment (Stml89) [inline hw_12/1.py]:
```suggestion
        deck.append(Card('Joker', 'Red'))
        deck.append(Card('Joker', 'Black'))
```

File: hw_12/2.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    def convert(cls, from_currency, amount, to_currency='BYN'):
+        """Конвертер валюты"""
+        if from_currency == to_currency:
+            return round(amount, 2), to_currency
+        if (from_currency, to_currency) in cls.exchange_rates:
+            rate = cls.exchange_rates[(from_currency, to_currency)]
+            return round(amount * rate, 2), to_currency
+        if (to_currency, from_currency) in cls.exchange_rates:
+            rate = cls.exchange_rates[(to_currency, from_currency)]
+            return round(amount / rate, 2), to_currency
```
Comment (Stml89) [inline hw_12/2.py]: it looks like it does the same as the previous `if` condition. am I right?

File: hw_14/1.py
Comment (Stml89) [inline hw_14/1.py:1]: not really good practice to have empty file in the repository

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hw_14/1.py
Comment (ETsiunel) [reply hw_14/1.py:1]: oops, I committed accidentally

File: hw_12/1.py (hunk 1 above)
Comment (ETsiunel) [reply hw_12/1.py]: fixed

File: hw_12/2.py (hunk 2 above)
Comment (ETsiunel) [reply hw_12/2.py]: Yes exactly :((
I was tired and didn't notice
Fixed

**Comment by ETsiunel** [comment]
All comments were fixed
@Stml89

**Comment by Stml89** [comment]
> All comments were fixed @Stml89

Navigate me, because can not find fixes

**Comment by ETsiunel** [comment]
so sorry, smth went wrong :) now all fixes are here

**Review by Stml89 — APPROVED** [review]

**Pull request merged**